hawkbit-repository/hawkbit-repository-jpa/src/main/java/org/eclipse/hawkbit/repository/jpa/JpaDeploymentManagement.java

```diff
@@ -182,33 +182,41 @@ protected JpaDeploymentManagement(final EntityManager entityManager, final Actio
                 .map(entry -> DeploymentManagement.deploymentRequest(entry.getKey(), entry.getValue()).build())
                 .collect(Collectors.toList());
 
-        return assignDistributionSets(deploymentRequests, null, offlineDsAssignmentStrategy);
+        return assignDistributionSets(tenantAware.getCurrentUsername(), deploymentRequests, null,
```

Comment:
I think `assignDistributionSets(deploymentRequests, null, offlineDsAssignmentStrategy)` should work as well